jenkins/opensearch-dashboards/Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
         buildDiscarder(logRotator(artifactNumToKeepStr: '1'))
     }
     stages {
-        stage('detect Docker image + args to use for the build') {
+        stage('detect') {
```

Comment:
Can you say more? Like `Detect CI image` or something else. Just detect is confusing for external folks

Reply:
I really just wanted to lowercase and shorten it, changed to `detect docker image + args`.